pytest_splunk_addon/standard_lib/data_models/Authentication.json

```diff
@@ -1,145 +1,238 @@
+                    "type": "optional",
+                    "comment": "The unique identifier or event code of the event signature."
+                },
+                {
+                    "name": "src",
+                    "type": "optional",
+                    "comment": "The name of the user involved in the event, or who initiated the event. For authentication privilege escalation events, this should represent the user targeted by the escalation."
+                },
+                {
+                    "name": "user",
+                    "type": "required",
+                    "comment": "The name of the user involved in the event, or who initiated the event. For authentication privilege escalation events, this should represent the user targeted by the escalation."
+                },
+                {
+                    "name": "user_id",
+                    "type": "optional",
+                    "comment": "The unique id of the user involved in the event."
+                },
+                {
+                    "name": "reason",
+                    "type": "optional",
+                    "validity": "if(action in ['success', 'failure'], action, null())",
+                    "comment": "The human-readable message associated with the authentication action (success or failure)."
+                },
+                {
+                    "name": "src_user",
+                    "condition": "src_user=*",
+                    "type": "conditional",
+                    "comment": "In privilege escalation events, src_user represents the user who initiated the privilege escalation. This field is unnecessary when an escalation has not been performed. If present it must be a valid user."
+                },
+                {
+                    "name": "vendor_account",
+                    "type": "optional",
+                    "comment": "The account that manages the user that initiated the request. The account represents the organization, a Cloud customer, or a Cloud account."
+                }
+            ],
+            "child_dataset": [
+                {
+                    "name": "Failed_Authentication",
+                    "tags": [
+                        [
+                            "authentication"
+                        ]
+                    ],
+                    "fields_cluster": [],
+                    "fields": [],
+                    "child_dataset": [],
+                    "search_constraints": "action=\"failure\""
+                },
+                {
+                    "name": "Successful_Authentication",
+                    "tags": [
+                        [
+                            "authentication"
+                        ]
+                    ],
+                    "fields_cluster": [],
+                    "fields": [],
+                    "child_dataset": [],
+                    "search_constraints": "action=\"success\""
+                },
```

Comment:
These 2 child datasets are missing from CIM official documentation, why they are here?

Reply:
So this is not a change in 5.3.2 they are not present in 4.15 as well, but they were added regardless, not sure why

Reply:
I assume they are used in content. created https://splunk.atlassian.net/browse/CIM-1244 to get them added after we verify that is the case.

Reply:
Again, the CIM docs are not the source of (complete) truth. The CIM json https://cd.splunkdev.com/EnterpriseSecurity/sa-commoninformationmodel/-/blob/develop/packages/sa-commoninformationmodel/src/main/resources/splunk/default/data/models/Authentication.json 
lists the object 
`"objectName": "Successful_Authentication",
`
with the above constraint `"search": "action=\"success\""`

The reason it is not listed in the CIM Docs is because this object (dataset) type does not have a set of cim tags and cim fields to be extracted for it. It only has a search constraint to define it.